variables.tf

```diff
@@ -32,6 +32,37 @@ variable "karpenter" {
   }
 }
 
+variable "enable_datadog" {
+  description = "Enable Datadog integration"
+  type        = bool
+  default     = false
+}
+
+variable "datadog" {
+  description = "Map of Datadog configurations"
+  type        = any
+  default     = {}
+  # TODO: currently possible values. How to document them?
+  # agent_manifest_override
+  # agent_manifest_features
+  # agent_api_key_name
+  # agent_app_key_name
+  # operator_chart_version
+}
+
+variable "datadog_operator_values" {
+  description = "Map of Datadog Operator values"
+  type        = map(string)
+  default     = {}
+  # This needs to be a separated variable with type map(string) otherwise I could not do the merge in datadog.tf
+}
+
+variable "datadog_operator_sensitive_values" {
+  description = "Map of Datadog Operator sensitive values"
+  type        = map(string)
+  default     = {}
+}
+
```

Comment:
The fist variable is used in datadog.tf and is merged with a local. Unfortunately the only way I found to make this work was to create a separated variable and set the type to `map(string)`, otherwise with `any` it was always of type tuple